pkg/resourcemanager/mysql/server/reconcile.go

```diff
@@ -190,9 +190,9 @@ func (m *MySQLServerClient) Ensure(ctx context.Context, obj runtime.Object, opts
 			instance.Status.Provisioned = false
 			instance.Status.FailedProvisioning = true
 			return true, nil
-
```

Comment:
Above on line 179 it says: ""Postgres server exists but may not be ready" - wrong DB type, this is mySQL.

Maybe fix that here too?

Reply:
Oh, I had fixed that but accidentally removed it when I ripped out my logging - will redo.